idd/Energy+.idd.in

```diff
@@ -52927,6 +53038,337 @@ AirConditioner:VariableRefrigerantFlow,
+       \type real
+       \units C
+       \default 5.0
+       \note Enter the maximum outdoor temperature above which the crankcase heaters are disabled.
+  A10, \field Defrost Strategy
+       \type choice
+       \key ReverseCycle
+       \key Resistive
+       \default Resistive
+       \note Select a defrost strategy. 
+       \note Reverse cycle reverses the operating mode from heating to 
+       \note cooling to melt frost formation on the condenser coil. 
+       \note The resistive strategy uses a resistive heater
+       \note to melt the frost.
+  A11, \field Defrost Control
+       \type choice
+       \key Timed
+       \key OnDemand
+       \default Timed
+       \note Choose a defrost control type. 
+       \note Either use a fixed Timed defrost period or select
+       \note OnDemand to defrost only when necessary.
+  A12, \field Defrost Energy Input Ratio Modifier Function of Temperature Curve Name
+       \type object-list
+       \object-list LinearCurves
+       \object-list QuadraticCubicCurves
+       \object-list UniVariateTables
+       \note A valid performance curve must be used if ReverseCycle defrost strategy is selected.
+  N21, \field Defrost Time Period Fraction
+       \type real
+       \minimum 0.0
+       \default 0.058333
+       \note Fraction of time in defrost mode.
+       \note Only applicable if timed defrost control is specified.
+  N22, \field Resistive Defrost Heater Capacity
+       \type real
+       \minimum 0.0
+       \default 0.0
+       \autosizable
+       \units W
+       \note Enter the size of the resistive defrost heating element.
+       \note Only applicable if resistive defrost strategy is specified
+       \ip-units W
+  N23, \field Maximum Outdoor Dry-bulb Temperature for Defrost Operation
+       \type real
+       \units C
+       \default 5.0
+       \note Enter the maximum outdoor temperature above which the crankcase heaters are disabled.	
+  N24, \field Compressor maximum delta Pressure
+       \required-field
+       \type real 
+       \units kPa
+       \default 4500
+       \minimum 0.0
+       \maximum 50000
+  N25, \field Number of Compressor Loading Index Entries
+       \required-field
+       \type integer
+       \default 2
+       \minimum 2
+       \maximum 9
+       \note First index represents minimal capacity operation 
+       \note Last index represents full capacity operation 
+  N26, \field Compressor Speed at Loading Index 1 
+       \type real
+       \note Minimum compressor speed 
+       \note unit in Rev/Sec	   
+  A13, \field Loading Index 1 Evaporative Capacity Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BiQuadraticCurves
+       \object-list BiVariateTables
+  A14, \field Loading Index 1 Compressor Power Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BiQuadraticCurves
+       \object-list BiVariateTables
```

Comment:
Yes, these fields are required. Codes were designed in GetVRFInput() to ensure that.